src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/index.jsx

```diff
@@ -39,11 +77,26 @@ class SentryApplicationRow extends React.PureComponent {
           </SentryAppBox>
         </Flex>
 
-        <Flex>
+        <StyledButtonGroup>
           <Box>
-            <Button icon="icon-trash" onClick={() => {}} className={btnClassName} />
+            <StyledInstallButton
+              onClick={this.install}
```

Comment:
If someone has already installed this app on their organization - do we want this button to be grayed out or to say something else? (Might be something to discuss later on tbh)

Reply:
Yea, def should in the long-run. I'll see how easy it is to do that now.